src/modules/nav/nav.component.ts

```diff
@@ -43,6 +43,8 @@ export class StacheNavComponent implements OnInit, StacheNav {
 
     if (route.fragment) {
       extras.fragment = route.fragment;
+      let element = document.getElementById(route.fragment);
```

Comment:
Should probably check to make sure the element exists before attempting to use `scrollIntoView`, as it would throw an error.
```
if (element) { ... }
```